cmd/helm/show.go

```diff
@@ -151,6 +151,9 @@ func addShowFlags(subCmd *cobra.Command, client *action.Show) {
 	f := subCmd.Flags()
 
 	f.BoolVar(&client.Devel, "devel", false, "use development versions, too. Equivalent to version '>0.0.0-0'. If --version is set, this is ignored")
+	if subCmd.Name() == "values" {
+		f.StringVar(&client.JSONPathTemplate, "jsonpath", "", "use jsonpath output format")
```

Comment:
The help text here isn't very descriptive. A user unfamiliar with JSONPath might think this is a binary flag. Could the help text read something like:

> supply a JSONPath expression to filter the output

Reply:
ok! done;